content/id/docs/concepts/services-networking/service.md

```diff
@@ -0,0 +1,1065 @@
+{{% tab name="Baidu Cloud" %}}
+```yaml
+[...]
+metadata:
+    name: my-service
+    annotations:
+        service.beta.kubernetes.io/cce-load-balancer-internal-vpc: "true"
+[...]
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+
+#### Dukungan untuk SSL di AWS
+Dukungan parsial untuk SSL bagi kluster yang dijalankan di AWS mulai diterapkan, 
+mulai versi 1.3 terdapat 3 anotasi yang dapat ditambahkan pada `Service` dengan tipe 
+`LoadBalancer`:
+
+```yaml
+metadata:
+  name: my-service
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-ssl-cert: arn:aws:acm:us-east-1:123456789012:certificate/12345678-1234-1234-1234-123456789012
+```
+
+Anotasi pertama memberikan spesifikasi ARN dari sertifikat yang akan digunakan. 
+Sertifikat yang digunakan bisa saja berasal dari _third party_ yang diunggah ke IAM atau 
+sertifikat yang dibuat secara langsung dengan menggunakan sertifikat manajer AWS.
+
+```yaml
+metadata:
+  name: my-service
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-backend-protocol: (https|http|ssl|tcp)
+```
+
+Anotasi kedua memberikan spesifikasi bagi protokol yang digunakan oleh `Pod` untuk saling berkomunikasi.
+Untuk HTTPS dan SSL, ELB membutuhkan `Pod` untuk melakukan autentikasi terhadap dirinya sendiri melalui 
+koneksi yang dienkripsi.
+
+Protokol HTTP dan HTTPS akan memilih mekanisme _proxy_ di tingkatan ke-7: 
+ELB akan melakukan terminasi koneksi dengan pengguna, melakukan proses _parsing_ _headers_, serta 
+memasukkan _value_ bagi _header_ `X-Forwarded-For` dengan alamat IP pengguna (_Pod_ hanya dapat melihat 
+alamat IP dari ELB pada akhir koneksi yang diberikan) ketika melakukan _forwarding_ suatu _request_.
+ 
+Protokol TCP dan SSL akan memilih mekanisme _proxy_ pada tingkatan 4: ELB akan melakukan _forwarding_ trafik 
+tanpa melakukan modifikasi _headers_.
+
+Pada _environment_ campuran dimana beberapa _port_ diamankan sementara _port_ lainnya dalam kondisi tidak dienkripsi, 
+anotasi-anotasi berikut dapat digunakan:
+
+```yaml
+    metadata:
+      name: my-service
+      annotations:
+        service.beta.kubernetes.io/aws-load-balancer-backend-protocol: http
+        service.beta.kubernetes.io/aws-load-balancer-ssl-ports: "443,8443"
+```
+
+Pada contoh di atas, jika `Service` memiliki 3 buah _port_, yaitu: `80`, `443`, dan 
+`8443`, maka `443` adan `8443` akan menggunakan sertifikat SSL, tetapi `80` hanya akan 
+di-_proxy_ menggunakan protokol HTTP.
+
+Mulai versi 1.9, `Service` juga dapat menggunakan [_predefined_ _policy_](http://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-security-policy-table.html)
+untuk HTTPS atau _listener_ SSL. Untuk melihat _policy_ apa saja yang dapat digunakan, kamu dapat menjalankan perintah _awscli_:
+
+```bash
+aws elb describe-load-balancer-policies --query 'PolicyDescriptions[].PolicyName'
+```
+
+_Policy_ ini kemudian dapat dispesifikasikan menggunakan anotasi 
+"_service.beta.kubernetes.io/aws-load-balancer-ssl-negotiation-policy_", contohnya:
```

Comment:
service.beta.kubernetes.io/aws-load-balancer-ssl-negotiation-policy need to be in backtick